### DISCUSSION TITLE
When i run this command in wsl forge coverage  --fork-url $SEPOLIA_RPC_URL

### DISCUSSION THREAD

**Discussion by today988** [Cyfrin/foundry-full-course-cu#1373] [Q&A]
When i run this command in my  wsl terminal
forge coverage  --fork-url $SEPOLIA_RPC_URL  

then i found this,

Compiler run successful!
Analysing contracts...
Running tests...
The application panicked (crashed).
Message:  Unable to create fork:
Could not instantiate forked environment with fork url: https://eth-sepolia.g.alchemy.com/v2/3LVqu53YfOnZtMzdXv8CFQki7tTyJUjd
Location: crates/evm/core/src/backend/mod.rs:428

This is a bug. Consider reporting it at https://github.com/foundry-rs/foundry

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
Aborted

**Comment by note-xin** [comment]
### troubleshoot and resolve the issue ###

1. Test Connectivity: Test the connectivity to the fork URL using tools like `curl` or `wget` from your WSL terminal to ensure you can reach the specified endpoint without any issues.
2. Retry the Command: Sometimes, network issues or temporary problems with the forked environment can cause failures. Try running the `forge coverage` command again.

second one worked for me while running into another error.

**Reply by today988** [reply]
Thankyou,
It worked
